2015-07-14  Matthias Klose  <doko@ubuntu.com>

	PR target/66840
	* config/rs6000/t-rs6000 (TM_H): Add rs6000-cpus.def.

diff -Nrup a/gcc/config/rs6000/t-rs6000 b/gcc/config/rs6000/t-rs6000
--- /gcc/config/rs6000/t-rs6000	2013-08-14 05:55:11.000000000 -0600
+++ gcc/config/rs6000/t-rs6000	2018-04-18 12:09:30.614737081 -0600
@@ -19,6 +19,7 @@
 # <http://www.gnu.org/licenses/>.
 
 TM_H += $(srcdir)/config/rs6000/rs6000-builtin.def
+TM_H += $(srcdir)/config/rs6000/rs6000-cpus.def
 
 rs6000.o: $(CONFIG_H) $(SYSTEM_H) coretypes.h $(TM_H) \
   $(RTL_H) $(REGS_H) hard-reg-set.h \
